test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -10961,10 +10960,19 @@ def revert_to_defaults(self):
         """
         self.logger.info(self.logprefix +
                          "reverting configuration to defaults")
-        self.server.manager(MGR_CMD_DELETE,
-                            SCHED,
-                            id="@default")
         self.server.manager(MGR_CMD_LIST, SCHED)
+        tmpschd = self.server.schedulers.keys()
```

Comment:
So this deletes the schedulers from the server, but what about from the self.server.schedulers or self.sched dictionaries?

Reply:
When MGR_CMD_DELETE is issued and rc is 0 it is deleted from the dictionary . It is handled in a seperate block of MGR_CMD_DELETE .